pkg/networkgraph/tree/radix_test.go

```diff
@@ -128,6 +138,25 @@ func TestNRadixTreeIPv4Remove(t *testing.T) {
 	}
 }
 
+func TestNRadixTreeFindCIDR(t *testing.T) {
```

Comment:
This unit test doesn't actually trigger the crash if the change is not made. I need to find a case that actually triggers the crash.